erpnext/erpnext_integrations/doctype/stripe_settings/stripe_settings.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
```

Comment:
Already exists in frappe. If any particular code related to ERPNext, create a new file stripe.py under erpnext_integrations and add the ERPNext specific code.